INSTALL.md

```diff
@@ -13,7 +13,25 @@
 
 # Prequisites
 
-Since currently CLI runs all of it's tasks in [Docker](https://www.docker.com/), make sure it's [installed](https://docs.docker.com/get-docker/).
+Since currently CLI runs all of it's tasks either in [Docker](https://www.docker.com/) or [Podman](https://podman.io/), make sure one of these is installed.
```

Comment:
Maybe `via` and links to the sections??

```suggestion
Since currently CLI runs all of it's tasks either via [Docker](#docker) or [Podman](#podman), make sure one of these is installed.
```